tests/storage_nats_kv.rs

```diff
@@ -0,0 +1,156 @@
+use anyhow::Result;
+use std::collections::HashMap;
+
+mod common;
+use common::make_nats_instance;
+use semver::Version;
+
+use wadm::storage::{
```

Comment:
spacing here is inconsistent

Reply:
Ah will fix, meant to separate local module stuff and external deps!

- [ ] Fix grouping of external versus internal modules